### PR TITLE
feat: add recommendations for ecotone changes

### PR DESCRIPTION
**Pull request by smartcontracts**

### CODE REVIEW

**Review by sbvegan — APPROVED** [review]
Looks good to me!

File: pages/builders/notices/ecotone-fee-changes.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Understanding the Change
+
+The L1 Data Fee component of the transaction fee for OP Stack (and therefore OP Mainnet) is expected to change with the proposed Ecotone upgrade.
+This new formula is designed to account for the option to post transaction data within data blobs instead of as Ethereum calldata, a feature enabled by [EIP-4844](https://eip4844.com).
+Read the [Ecotone Formula](/stack/transactions/fees#ecotone) section of the [Transaction Fees](/stack/transactions/fees) page for more information about the new formula.
+
+## Preparing Your Application
+
+Changes to the L1 Data Fee formula may affect your application if you are computing this fee component on your own.
```
Comment (roninjin10) [inline pages/builders/notices/ecotone-fee-changes.mdx:23]: Should this be wrapped in a <Callout>?

**Review by roninjin10 — APPROVED** [review]

**Review by K-Ho — APPROVED** [review]
Amazing lets ship it!

File: pages/builders/notices/ecotone-fee-changes.mdx (hunk 1 above)
Comment (sbvegan) [reply pages/builders/notices/ecotone-fee-changes.mdx]: added

**Pull request merged**